### PR TITLE
QUA-179: Update Jira integration description

### PR DESCRIPTION
**Pull request by noelia-lencina**
https://codeclimate.atlassian.net/browse/QUA-179

### CODE REVIEW

**Comment by noelia-lencina** [comment]
CI error: I think it's because we are using the old configuration for Circle CI. I will fix it in another PR and rebase this.

File: lib/cc/services/jira.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 class CC::Service::Jira < CC::Service
   class Config < CC::Service::Config
     attribute :domain, Axiom::Types::String,
-      description: "Your JIRA host domain (e.g. yourjira.com:PORT, please exclude https://)"
+      description: "Your JIRA host domain (e.g. yourjira.com[:PORT], please exclude https://. Include PORT only if you are using a port number to access your own server)"
```
Comment (filipesperandio) [inline lib/cc/services/jira.rb:6]:
```suggestion
      description: "Your JIRA host domain (e.g. yourjira.com[:PORT], please exclude "https://" and only  add PORT if you are using a different port number to access your own server that is not the default)"
```

**Review by filipesperandio — APPROVED** [review]

File: lib/cc/services/jira.rb (hunk 1 above)
Comment (noelia-lencina) [reply lib/cc/services/jira.rb:6]: I think that even if you are using the default port, you need to include the port number here. You don't need the port number if you are not hosting your own Jira instance or if you are but have a DNS configured so you can access the instance just by its domain.

**Review by fede-moya — APPROVED** [review]

File: lib/cc/services/jira.rb (hunk 1 above)
Comment (filipesperandio) [reply lib/cc/services/jira.rb:6]: PORT follows its protocol, for https it is 443 - it is the only use I see, so if not specified, 443 is what it is gonna be, right?

Comment (filipesperandio) [reply lib/cc/services/jira.rb:6]: feel free to drop it if  it made things more confusing.  🤦

**Comment by filipesperandio** [comment]
Seems like we need to port this over to circle 2... too long we don't do anything here.

**Pull request merged**